core/src/main/java/org/sonarsource/sonarlint/core/issue/IssueServiceImpl.java

```diff
@@ -160,6 +163,43 @@ public CompletableFuture<Void> addComment(AddIssueCommentParams params) {
       .orElseGet(() -> addCommentOnServerIssue(configurationScopeId, issueKey, params.getText()));
   }
 
+  @Override
+  public CompletableFuture<Boolean> reopenIssue(ReopenIssueParams params) {
+    var issueId = params.getIssueId();
+    var issueUuid = UUID.fromString(issueId);
+    var localOnlyIssueStore = localOnlyIssueStorageService.get();
+    var configurationScopeId = localOnlyIssueStore.findConfigurationScopeIdByIssueId(issueUuid);
```

Comment:
In fact we could let the client provide the configuration scope ID here. It will be useful anyway later to reopen server issues